lib/src/Test/Pos/Helpers.hs

```diff
@@ -56,22 +56,25 @@ cborFlatTermValid = property . validFlatTerm . toFlatTerm . encode
 
 -- Test that serialized 'a' has canonical representation, i.e. if we're able to
 -- change its serialized form, it won't be successfully deserialized.
-cborCanonicalRep :: forall a. (Bi a, Show a) => a -> Property
+cborCanonicalRep :: forall a . (Bi a, Eq a, Show a) => a -> Property
 cborCanonicalRep a = property $ do
-    let sa = serialize a
-    sa' <- R.serialise <$> perturbCanonicity (R.deserialise sa)
-    let out = decodeFull @a $ sa'
-    pure $ case out of
-        -- perturbCanonicity may have not changed anything. Decoding can
-        -- succeed in this case.
-        Right a' -> counterexample (show a') (sa == sa')
-        -- It didn't decode. The error had better be a canonicity violation.
-        Left err -> counterexample (show err) (isCanonicityViolation err)
+  let sa = serialize a
```

Comment:
At least part of the change  here seems to be whitespace, which makes seeing the actual functional changes a little difficult. Also, the [style guide](https://github.com/input-output-hk/cardano-sl/blob/develop/docs/style-guide.md) for cardano-sl say "4 space indents". I use 2 space for my own projects, but here I go with 4.